site/reference/live/install-resource-group/README.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
we don't really need these hugo directives anymore, a simple title suffices. Should do a cleanup at some point.